proto/vector.proto

```diff
@@ -3,11 +3,11 @@ package vector;
 
 import "event.proto";
 
-message EventRequest {
-  event.EventWrapper message = 1;
+message PushEventsRequest {
+  repeated event.EventWrapper events = 1;
```

Comment:
Now that 0.14 has gone out with the gRPC vector protocol, does this need a workaround to avoid breaking backwards compatibility?

Reply:
That's a great point. I hadn't realized we already officially announced the new gRPC protocol.

Although having said that, both the [source](https://vector.dev/docs/reference/configuration/sources/vector/) and [sink](https://vector.dev/docs/reference/configuration/sinks/vector/) still default to v1 (e.g. the old TCP-based implementation), and neither document the `v2` switch to use gRPC, so maybe we're in the clear to make breaking changes here?

It'd at least be somewhat annoying to have to support both use-cases for something we only just shipped, and arent' actively using/promoting yet.

cc @binarylogic

Reply:
We haven't "officially" announced it yet, but it is available

Reply:
It does appear to be undocumented so I think it's ok to break compatibility here. We should just avoid suggesting to anyone that they use it until it is documented (and thus officially released).

Reply:
I'll go ahead and resolve this discussion. We'll ship this breaking change, without having to mark is a breaking, since we haven't promoted the `v2` version of this source/sink-combo yet, and the documentation doesn't mention it either.

People that have found this themselves without any docs are likely technical enough to figure out what has changed.